controllers/host/host_suite_test.go

```diff
@@ -1,5 +1,5 @@
-/* SPDX-License-Identifier: Apache-2.0 */
-/* Copyright(c) 2019-2022 Wind River Systems, Inc. */
+// /* SPDX-License-Identifier: Apache-2.0 */
+// /* Copyright(c) 2019-2022 Wind River Systems, Inc. */
```

Comment:
`//` can be removed and `/*` be retained.
Also please update the copyright notice -> `2019-2024`

Reply:
Resolved the comment